site/content/blog/2021-06-01-whats-new-in-svelte-june-2021.md

```diff
@@ -0,0 +1,55 @@
+---
+title: What's new in Svelte: June 2021
+description: Progress on SvelteKit V1 and tighter TypeScript/Svelte integrations in language tools
+author: Daniel Sandoval
+authorURL: https://desandoval.net
+---
+
+This month, we saw lots of contributions to SvelteKit and its docs. The language tools also got some new features alongside some hydration bug fixes in Svelte core. Let's jump into the updates...
+
+## New in SvelteKit
```

Comment:
Some of the bigger features:
* `adapter-netlify` got a fix https://github.com/sveltejs/kit/pull/1467 and new documentation in the readme https://github.com/sveltejs/kit/tree/master/packages/adapter-netlify
* support for https keypair https://github.com/sveltejs/kit/pull/1456
* now bundling Vite with SvelteKit and using an upgraded version. Remove Vite from your `package.json` if it's there
* etags for binary responses https://github.com/sveltejs/kit/pull/1382
* Renamed `$layout` to `__layout` and `$error` to `__error`
* Removed `getContext` in favor of `request.locals`
* Renamed `.svelte` output directory to `.svelte-kit`. Update your `.gitignore` accordingly

Reply:
Great! Split these up between "new" and "bug fixes"